public/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Have you duly tested your website's responsiveness on a small screen? This meta tag states that you have. If that's not the case, you're preventing the user to zoom, which might be a terrible UX if the scale is improper.